docker/docker-compose.yml

```diff
@@ -36,6 +38,7 @@ services:
 
   nodejs:
     image: fidals/stb-nodejs:dev
+    command: gulp watch
```

Comment:
it's not clear if we should do build or watch by default.
So, the best way is to rm this command and add related recipes to Makefile.
We'll enforce explicitness in this case